administrator/components/com_content/helpers/content.php

```diff
@@ -114,4 +114,26 @@ public static function countItems(&$items)
 
 		return $items;
 	}
+
+	/**
+	 * This function returns the id from the table __content based on title passed as parameter
+	 * 
+	 * @param   string  $item  title
+	 *
+	 * @return   integer
+	 */
+	public static function getarticleid($item)
+	{
+		$db = JFactory::getDbo();
+		$query = $db->getQuery(true);
+		$query
+			->select('id')
+			->from($db->qn('#__content'))
+			->where($db->qn('title') . " = " . $db->q($item));
```

Comment:
As I have said it's not good idea to get article id using title. Use my above suggestion to get it.